src/hotspot/os/linux/cgroupV1Subsystem_linux.cpp

```diff
@@ -301,8 +301,8 @@ void CgroupV1MemoryController::print_version_specific_info(outputStream* st, jul
   jlong kmem_max_usage = kernel_memory_max_usage_in_bytes();
 
   OSContainer::print_container_helper(st, kmem_usage, "kernel_memory_usage_in_bytes");
-  OSContainer::print_container_helper(st, kmem_limit, "kernel_memory_max_usage_in_bytes");
-  OSContainer::print_container_helper(st, kmem_max_usage, "kernel_memory_limit_in_bytes");
+  OSContainer::print_container_helper(st, kmem_limit, "kernel_memory_limit_in_bytes");
```

Comment:
Looking at the other printouts in `os::Linux::print_container_info(...)` the limit is printed first, followed by usage and max usage. I think it would be good to use that order here as well to be more consistent.